batch-setup/make_meta_tiles.py

```diff
@@ -433,6 +458,12 @@ def render(self, num_retries, lense):
     parser.add_argument('--allowed-missing-tiles', default=2, type=int,
                         help='The maximum number of missing metatiles allowed '
                         'to continue the build process.')
+    parser.add_argument('--use-tiles-coords-generator', type=lambda x: bool(strtobool(x)), nargs='?',
```

Comment:
You can use action=store_true here instead of this. See argparse docs (https://docs.python.org/3/library/argparse.html#action) for more details.

Reply:
> You can use action=store_true here instead of this. See argparse docs (https://docs.python.org/3/library/argparse.html#action) for more details.

yup, looks like the current implementation will support multiple ways to set args.use_tiles_coords_generate to `True`
such as

--use-tiles-coords-generator

--use-tiles-coords-generator True

--use-tiles-coords-generator true

--use-tiles-coords-generator 1

--use-tiles-coords-generator yes